### PR TITLE
Making relative paths absolute

### PR DESCRIPTION
**Pull request by NickBusey**

### CODE REVIEW

**Comment by NickBusey** [comment]
Cleaned up version of https://github.com/meteor/todos/pull/215

Working on a complimentary guide PR now.

**Comment by NickBusey** [comment]
Sorry for making 2 PR's, should have just cleaned up the first one. Won't do it again ;)

File: imports/api/lists/server/publications.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 import { Meteor } from 'meteor/meteor';
 
-import { Lists } from '../lists.js';
+import { Lists } from '/imports/api/lists/lists.js';
```
Comment (abernix) [inline imports/api/lists/server/publications.js]: This is where this all-or-mostly absolute approach becomes less beneficial to me.  A single-level, relative traversal within the scope of the `lists` API directory is perfectly reasonable and much more portable, so I'd be in favor of not doing this (or anything like it).

You could easily move this `lists` API to another project with a different base structure without this change.

I get the intention of this, but I think it's important to highlight the strengths of multiple options and where they each have their strengths.

Comment (NickBusey) [inline imports/api/lists/server/publications.js]: Ok. This is why I asked beforehand if it was useful. Feel free to close.

**Comment by hwillson** [comment]
Hi @NickBusey - thanks for taking the time to submit this PR. Based on the discussion that came out of this and #215, It sounds like the best course of action for now is to leave the relative import paths in place. I'll close this for now - thanks again!

**Pull request closed without merging**